lib/helper/TestCafe.js

```diff
@@ -1208,7 +1208,7 @@ async function proceedClick(locator, context = null) {
     await assertElementExists(els, locator, 'Clickable element');
   }
 
-  const firstElement = await els.nth(0);
+const firstElement = await els.filterVisible().nth(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 0.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5059498)